runtime/kusama/src/lib.rs

```diff
@@ -455,7 +455,12 @@ parameter_types! {
 	pub const TipCountdown: BlockNumber = 1 * DAYS;
 	pub const TipFindersFee: Percent = Percent::from_percent(20);
 	pub const TipReportDepositBase: Balance = 1 * DOLLARS;
-	pub const TipReportDepositPerByte: Balance = 1 * CENTS;
+	pub const DataDepositPerByte: Balance = 1 * CENTS;
+	pub const BountyDepositBase: Balance = 1 * DOLLARS;
+	pub const BountyDepositPayoutDelay: BlockNumber = 3 * DAYS;
+	pub const BountyValueMinimum: Balance = 20 * DOLLARS;
+	pub const BountyDuration: BlockNumber = 14 * DAYS;
+	pub const MaximumReasonLenght: u32 = 16384;
```

Comment:
I don't have strong opinions on these constants but it seems like something that should bear further review.

cc @gavofyork @rrtti